fractions/opentracing/src/main/resources/modules/io/opentracing/main/module.xml

```diff
@@ -0,0 +1,14 @@
+<module xmlns="urn:jboss:module:1.3" name="io.opentracing">
+  <resources>
```

Comment:
Right, the user needs to bring a concrete implementation for the Tracer to have something useful, but this won't break in absence of a Tracer, as a `NoopTracer` is provided by the API.